meta-android/recipes-core/busybox/files/mdev-partname.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Script for mdev's uevent, which will create by-partname symlinks (like udev does)
+
+case "$ACTION" in
+	add|"")
+		ACTION="add"
```

Comment:
In which case is ACTION empty and is the value set here used somewhere later?

Reply:
Ah, yes, that's a line from upstream's mdev-mounts.sh I took as a basis. It's probably completely useless here -- as well as in the upstream's script...